cmd/template/cluster/flag.go

```diff
@@ -121,11 +122,21 @@ func (f *flag) Validate() error {
 	if !aws.ValidateRegion(f.Region) {
 		return microerror.Maskf(invalidFlagError, "--%s must be valid region name", flagRegion)
 	}
-	if f.MasterAZ == "" {
-		return microerror.Maskf(invalidFlagError, "--%s must not be empty", flagMasterAZ)
+
+	// AZ name(s)
+	if len(f.MasterAZ) != 1 && len(f.MasterAZ) != 3 {
```

Comment:
we might wanna try run 3 masters with 1 or 2 AZs, but this is probabbly fine for first wave of testing

Reply:
Yeah, for those who want that, I would suggest to edit the created manifest manually.